geode-core/src/main/java/org/apache/geode/management/internal/cli/converters/ConnectionEndpointConverter.java

```diff
@@ -102,47 +102,4 @@ public boolean getAllPossibleValues(List<Completion> completions, Class<?> targe
 
     return completions.size() > 0;
   }
-
-  public static void main(String[] args) {
```

Comment:
Wooo!  Too many of off these old, forgotten artifacts lying around.  Good to see them gone.